R/vad.R

```diff
@@ -0,0 +1,183 @@
+globalVariables(c("DBZH","VRADH","RHOHV"))
+NULL
+
+#' Create a Velocity Azimuth Display (VAD) plot
+#'
+#' A velocity azimuth display plot visualized the radial velocity (`VRAD`) as a function of the azimuth from the radar.
+#' Among others it can be used to asses the fit of the movement speeds in a vertical profile.
+#' For example, when the movement is not uniform but rather in multiple directions or rotating this will be visible as
+#' deviations from the sine function.
+#'
+#' @param x A polar volume from which range gates are extracted.
+#' @param ... Currently not used.
+#' @param vp A vertical profile to annotate the VAD plot with the fit in the vertical profile
+#' @param range The distance range in to filter the range gates with.
```

Comment:
other functions in bioRad use `range_min` and `range_max`, so I suggest using the same naming convention. `range` is more used for the continuous variable of radar range, not to set upper/lower limits

Reply:
changed